### PR TITLE
Improve golden record name selection

### PR DESCRIPTION
**Pull request by pudo**
Try to select more culturally appropriate names for entities which have multiple names.

### CODE REVIEW

File: tests/types/test_names.py [hunk 1]
```diff
@@ -19,6 +19,8 @@ def test_pick(self):
         self.assertEqual(names.pick(values), "Banana")
         self.assertEqual(names.pick("Banana"), "Banana")
         self.assertEqual(names.pick([]), None)
+        values = ["Robert Smith", "Rob Smith", "Robert SMITH"]
```
Comment (Rosencrantz) [inline tests/types/test_names.py:22]: Boys. Don't. Cry

File: followthemoney/types/name.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 from banal import ensure_list, first
 from normality import slugify
 from normality.cleaning import collapse_spaces, strip_quotes
-from fuzzywuzzy import fuzz
+from fuzzywuzzy import fuzz  # type: ignore
```
Comment (Rosencrantz) [inline followthemoney/types/name.py:4]: @pudo Can we just remove this, or is it being used elsewhere outside of these changes?

**Review by Rosencrantz — APPROVED** [review]

File: followthemoney/types/name.py (hunk 2 above)
Comment (pudo) [reply followthemoney/types/name.py:4]: We might want to ask @mynameisfiber about that. FtM used to use just `python-Levenshtein` but now we've adopted `fuzzywuzzy` which seems to wrap `python-Levenshtein`. I'm not quite sure what the value add is.

Comment (mynameisfiber) [reply followthemoney/types/name.py:4]: It's being used in `NameType.compare()`. It uses `fuzz.WRatio` which has heuristics to apply differently weighted levenshtein-like distances depending on the size difference of the strings. It makes a huge difference when dealing with "Joe Shmo" vs "Joe Jonathan Shmo the Third" or other partial prefix/suffix matches.

Comment (Rosencrantz) [reply followthemoney/types/name.py:4]: Thanks for confirming @mynameisfiber

**Pull request merged**